app/Main.hs

```diff
@@ -28,39 +127,40 @@ instance Read ValidatorHash where
   readsPrec _ x = [(fromString x, "")]
 
 instance ParseRecord ValidatorHash where
-    parseRecord = fmap getOnly parseRecord
+  parseRecord = fmap getOnly parseRecord
 instance ParseField ValidatorHash
 instance ParseFields ValidatorHash
 
 data Options = Options
-  { alwaysSucceedOutput              :: FilePath
```

Comment:
Why remove all the space formatting here?

Reply:
This happens automatically when you run the formatter (fourmolu, see my comment below). I don't think there'd be any way to have these kind of spaces automatically though as I'm not sure what settings you could have for that as they would vary a good bit I guess.

I guess we could just not use the formatter but I think it's nice to have the consistency if possible. What do you think?

Reply:
Maybe @jfischoff can help. @jfischoff can you advise me what formatting styles and standards you were using?